src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-coior: #00acdc;
```

Comment:
There seems to be a typo in the variable name `$button-coior`. It should be `$button-color` to correctly represent its purpose and to match the naming convention.